src/jdk.jlink/share/classes/jdk/tools/jlink/internal/plugins/SystemModulesPlugin.java

```diff
@@ -531,11 +520,8 @@ static class SystemModulesClassGenerator {
         private final int MH_VAR         = 1;  // variable for ModuleHashes
         private int nextLocalVar         = 2;  // index to next local variable
 
-        // Method visitor for generating the SystemModules::modules() method
-        private MethodVisitor mv;
-
         // name of class to generate
```

Comment:
With the switch the ClassDesc then some of the comments might need to be adjusted.